vinodtasks/Closure.js

```diff
@@ -0,0 +1,87 @@
+// function APP(name){
+//     function app(age){
+//         console.log(`my name is ${name} and age ${age} `)
+
+//     }
+//     app(23)
+// }
+// APP("vinod")
+
+// function outer(){
+//     let username="vinod"
+
+//     function inner(){
+//         return username;
+//     }
+//   return  inner
+
+
+// }
+// let fun =outer();
+
+
+// let val=fun();
+// console.log(val)
+
+// num=5
+// const addtwo=(num)=>{
+//     return num
+// }
+// console.log(addtwo(5))
+
+
+// addtwoo(5)
+// const addtwoo=function(num){
+//     return num+2
+// }
+
+// console.log(addtwo(5))
+// function addtwo(num){
+//     return num*5
+// }
+
```

Comment:
Uncomment this code and use proper syntax for all examples

Reply:
uncomment the code proper examples